### DISCUSSION TITLE
Control Center should be renamed Quick Settings Panel

### DISCUSSION THREAD

**Discussion by fredriklundholm** [valinet/ExplorerPatcher#2679] [General]
I was looking over the translations and tried to make out what "Control Centre" meant.
Was it a mis-spelt Control Panel? No in fact on W11 "Control Centre" Summons the Quick Settings Panel. 
Would it be appropriate to Change the occurances of "Control Centre" to "Quick Settings Panel" in ep_gui.en-US.rc?

**Comment by pyrates999** [comment]
Good idea.

**Comment by tallman69** [comment]
I have seen that button called Control Center, Action Center, and Quick Settings.  Officially Microsoft calls it the Control Center.  That is the name you get if you hover over it with the pointer.

**Reply by fredriklundholm** [reply]
Dear Tallman: 
The official name seems to be Quick Settings:
https://support.microsoft.com/en-us/windows/how-to-open-notification-center-and-quick-settings-f8dc196e-82db-5d67-f55e-ba5586fbb038
I counld not find any hit on the microsoft website of "Control Centre". 
Maybe it is an internal name.

EDJT: Tallman, the name "Control Centre" is the name given to the Icon/flyout when using explorer patcher! In regular Windows this icon does not exist.

**Comment by tallman69** [comment]
@fredriklundholm 
Yes, you are correct.  I thought that icon first appeared when I first installed Widows 11, but I was wrong.  Thank you for the correction.  Changing the name would be a good idea.  The name Control Center is too close to the name Control Panel.
